### DISCUSSION TITLE
Install infinite loop with aws "extra"

### DISCUSSION THREAD

**Discussion by 1sttec** [PrefectHQ/prefect#4905] [General]
python -m pip install prefect[all_extras]
  
or any combination of extras that includes 'aws' hangs on infinite loop with repeating msg:
  
>  "WARNING: dask-cloudprovider X.X.X does not provide the extra 'aws'"
  
- Windows Server 2012 R2
- Python 3.7.1
- Pip 21.2.4
  
Please advise.

**Comment by zanieb** [comment]
Please do not double open issues. I will respond in #4906
